src/util/keyutil.ts

```diff
@@ -0,0 +1,137 @@
+import * as React from "react";
+import * as electron from "electron";
+
+type KeyPressDecl = {
+    mods: Record<ModKeyStrs, boolean>;
+    key: string;
+};
+
+function parseKeyDescription(keyDescription: string): KeyPressDecl {
+    let rtn = { key: "", mods: {} } as KeyPressDecl;
+    rtn.mods.Cmd = false;
```

Comment:
is it ugly to init this way? wasn't sure how to properly init a record type
seems I have to set each of the values otherwise they end up as none